src/storage/wal/wal_record.cpp

```diff
@@ -78,10 +78,51 @@ StorageStructureID StorageStructureID::newAdjColumnID(
     return retVal;
 }
 
+string walRecordTypeToString(WALRecordType walRecordType) {
+    switch (walRecordType) {
+    case WALRecordType::PAGE_UPDATE_OR_INSERT_RECORD: {
+        return "PAGE_UPDATE_OR_INSERT_RECORD";
+    }
+    case WALRecordType::TABLE_STATISTICS_RECORD: {
+        return "TABLE_STATISTICS_RECORD";
+    }
+    case WALRecordType::COMMIT_RECORD: {
+        return "COMMIT_RECORD";
+    }
+    case WALRecordType::CATALOG_RECORD: {
+        return "CATALOG_RECORD";
+    }
+    case WALRecordType::NODE_TABLE_RECORD: {
+        return "NODE_TABLE_RECORD";
+    }
+    case WALRecordType::REL_TABLE_RECORD: {
+        return "REL_TABLE_RECORD";
+    }
+    case WALRecordType::OVERFLOW_FILE_NEXT_BYTE_POS_RECORD: {
+        return "OVERFLOW_FILE_NEXT_BYTE_POS_RECORD";
+    }
+    case WALRecordType::COPY_NODE_CSV_RECORD: {
+        return "COPY_NODE_CSV_RECORD";
+    }
+    case WALRecordType::COPY_REL_CSV_RECORD: {
+        return "COPY_REL_CSV_RECORD";
+    }
+    case WALRecordType::DROP_TABLE_RECORD: {
+        return "DROP_TABLE_RECORD";
+    }
+    case WALRecordType::DROP_PROPERTY_RECORD: {
+        return "DROP_PROPERTY_RECORD";
+    }
+    default: {
+        assert(false);
```

Comment:
Better to throw an exception here instead of asserting. (I think we should gradually remove the usage of assert in switch cases)

Reply:
I think we should never reach this code unless we made some errors in our switch/case. (eg. missing return). I think it is better to use assertion here.